Bottom: 94a1dd456de627cdff21f41cc91293e2e14f8f02
Top:    fc4c8428c3a0f994eabd7d26239f106f030611e5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-07 19:40:05 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
